lib/need_for_type/display_window.rb

```diff
@@ -3,6 +3,15 @@
 
 module NeedForType
   class DisplayWindow < Window
+    @@need_for_type_ascii_art =
+" _   _               _    __             _______               
+| \\ | |             | |  / _|           |__   __|              
+|  \\| | ___  ___  __| | | |_ ___  _ __     | |_   _ _ __   ___ 
+| . ` |/ _ \\/ _ \\/ _` | |  _/ _ \\| '__|    | | | | | '_ \\ / _ \\
+| |\\  |  __/  __/ (_| | | || (_) | |       | | |_| | |_) |  __/
+|_| \\_|\\___|\\___|\\__,_| |_| \\___/|_|       |_|\\__, | .__/ \\___|
+                                               __/ | |         
+                                              |___/|_|"
```

Comment:
http://patorjk.com/software/taag/#p=display&f=Slant&t=Need%20for%20Type perhaps this one is better 😄